test/HttpApiHandlerTests.cs

```diff
@@ -73,6 +73,78 @@ private void CreateCheckStatusResponse_Throws_Exception_When_NotificationUrl_Mis
+                checkStatus.TerminatePostUri);
+        }
+
+        [Fact]
+        public void CreateCheckStatus_Returns_Corrent_CheckStatus_based_on_custom_connection_value()
+        {
+            var httpApiHandler = new HttpApiHandler(new DurableTaskExtension() { NotificationUrl = new Uri(TestConstants.NotificationUrl) }, null);
+            CheckStatus checkStatus = httpApiHandler.CreateCheckStatus(TestConstants.InstanceId, null, TestConstants.CustomConnectionName);
+            Assert.NotNull(checkStatus);
+            Assert.Equal(checkStatus.Id, TestConstants.InstanceId);
+            Assert.Equal(
+                "http://localhost:7071/admin/extensions/DurableTaskExtension/instances/7b59154ae666471993659902ed0ba742?taskHub=DurableFunctionsHub&connection=TestConnection&code=mykey",
+                checkStatus.StatusQueryGetUri);
+            Assert.Equal(
+                "http://localhost:7071/admin/extensions/DurableTaskExtension/instances/7b59154ae666471993659902ed0ba742/raiseEvent/{eventName}?taskHub=DurableFunctionsHub&connection=TestConnection&code=mykey",
+                checkStatus.SendEventPostUri);
+            Assert.Equal(
+                "http://localhost:7071/admin/extensions/DurableTaskExtension/instances/7b59154ae666471993659902ed0ba742/terminate?reason={text}&taskHub=DurableFunctionsHub&connection=TestConnection&code=mykey",
+                checkStatus.TerminatePostUri);
+        }
+
+        [Fact]
+        public void CreateCheckStatus_Returns_Corrent_CheckStatus_based_on_custom_values()
+        {
+            var httpApiHandler = new HttpApiHandler(new DurableTaskExtension() { NotificationUrl = new Uri(TestConstants.NotificationUrl) }, null);
+            CheckStatus checkStatus = httpApiHandler.CreateCheckStatus(TestConstants.InstanceId, TestConstants.TaskHub, TestConstants.CustomConnectionName);
+            Assert.NotNull(checkStatus);
+            Assert.Equal(checkStatus.Id, TestConstants.InstanceId);
+            Assert.Equal(
+                "http://localhost:7071/admin/extensions/DurableTaskExtension/instances/7b59154ae666471993659902ed0ba742?taskHub=SampleHubVS&connection=TestConnection&code=mykey",
+                checkStatus.StatusQueryGetUri);
+            Assert.Equal(
+                "http://localhost:7071/admin/extensions/DurableTaskExtension/instances/7b59154ae666471993659902ed0ba742/raiseEvent/{eventName}?taskHub=SampleHubVS&connection=TestConnection&code=mykey",
+                checkStatus.SendEventPostUri);
+            Assert.Equal(
+                "http://localhost:7071/admin/extensions/DurableTaskExtension/instances/7b59154ae666471993659902ed0ba742/terminate?reason={text}&taskHub=SampleHubVS&connection=TestConnection&code=mykey",
+                checkStatus.TerminatePostUri);
+        }
+

```

Comment:
It's great to have these unit tests to help ensure we don't accidentally break the URL formats - thanks for adding them!

Reply:
Sure. Tests are super useful to validate all expected behaviors.